styles.css

```diff
@@ -8,111 +8,72 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    padding: 80px 165px;
     gap: 30px;
     background: #FFFFFF;
-    width: 1440px;
-    height: 332px;
+    height: 335px;
+    justify-content: center;
+    width: 100%;
 }
+
 .stats-section-firstcolumn {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    padding: 0px;
     gap: 8px;
-    width: 540px;
-    height: 168px;
-
-
 }
 .stats-firstcolumn-heading {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    padding: 0px;
     gap: 8px;
     color: #18191F;
     width: 540px;
-    height: 128px;
+    height: 130px;
```

Comment:
Make it max width and remove height. Instead, add line height to the heading which is having two lines, which will make the height of the container automatically.